### PR TITLE
Use new settings registration method

### PR DESCRIPTION
**Pull request by tomneedham**

### CODE REVIEW

**Comment by PVince81** [comment]
"Monitoring" ? I would never have found it if I didn't see this section appearing after enabling the app.

**Comment by PVince81** [comment]
Should we use "Troubleshooting" or add it inside a "Help" or "Get Help" section along with documentation ?

File: lib/AdminPanel.php [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?php
+
 /**
- * @author Felix Boehm <felix@owncloud.com>
- * @author Jörn Friedrich Dreyer <jfd@butonic.de>
+ * @author Tom Needham <tom@owncloud.com>
  *
- * @copyright Copyright (c) 2016, ownCloud GmbH.
+ * @copyright Copyright (c) 2017, ownCloud, GbmH.
```
Comment (mmattel) [inline lib/AdminPanel.php]: Gbmh -> Gmbh

Comment (DeepDiver1975) [inline lib/AdminPanel.php]: and no , and no .

**Comment by PVince81** [comment]
:+1:

**Pull request merged**